main.tex

```diff
@@ -275,12 +276,12 @@
 \begin{lstlisting}
 https://github.com/cosmos/cosmos/blob/master/WHITEPAPER.md
 \end{lstlisting}
-we shall use the exact object itself:
+we use the object itself:
 \begin{lstlisting}
 Qme4z71Zea9xaXScUi6pbsuTKCCNFp5TAv8W5tjdfH7yuH
 \end{lstlisting}
 
-By using content addresses to build the knowledge graph we gain \linkred{https://steemit.com/web3/@hipster/an-idea-of-decentralized-search-for-web3-ce860d61defe5est}{the so much needed} \linkgreen{https://ipfs.io/ipfs/QmV9tSDx9UiPeWExXEeH6aoDvmihvx6jD5eLb4jbTaKGps}{IPFS} - \linkgreen{https://ipfs.io/ipfs/QmXHGmfo4sjdHVW2MAxczAfs44RCpSeva2an4QvkzqYgfR}{like} superpowers of p2p protocols that are desired for a search engine:
+By using content addresses to build the knowledge graph we gain \linkred{https://steemit.com/web3/@hipster/an-idea-of-decentralized-search-for-web3-ce860d61defe5est}{the so much needed} \linkgreen{https://ipfs.io/ipfs/QmV9tSDx9UiPeWExXEeH6aoDvmihvx6jD5eLb4jbTaKGps}{IPFS} - \linkgreen{https://ipfs.io/ipfs/QmXHGmfo4sjdHVW2MAxczAfs44RCpSeva2an4QvkzqYgfR}{like} superpowers of P2P protocols that are desired for a search engine:
```

Comment:
WHY CAPS?

Reply:
Its an abbreviation. We spoke about making abbreviation using caps for all texts with you

Reply:
changed